src/main/java/org/ssu/edu/teachua/ui/pages/challenges/AddChallengePage.java

```diff
@@ -1,10 +1,104 @@
 package org.ssu.edu.teachua.ui.pages.challenges;
 
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
 import org.ssu.edu.teachua.ui.base.BasePage;
 
 public class AddChallengePage extends BasePage {
+    @FindBy(how = How.XPATH, using = ".//input[@id='sortNumber']")
```

Comment:
This XPATH should not start with `.` (Only in classes inherited by BaseComponent)